bbc/simorgh
# [WSTEAM1-330] Delete references to legacy colours in Psammead

Resolves JIRA [WSTEAM1-330] as part of [WSTEAM1-326]

Overall change
======
Removes references to legacy colours in `legacy/psammead` and replaces them with colours from the `ThemeProvider` palette.

Code changes
======
- Replaces all legacy colour references in `legacy/psammead/*` with colours from the `ThemeProvider` palette
- Deletes `colours.js` and `colours.test.js` files
- Update unit tests
- Update stories

Testing
======
- Pull down branch
- Check that there are no references to `src/colours` within `legacy/psammead/` folders

Helpful Links
======
_Add Links to useful resources related to this PR if applicable._

[Coding Standards](https://github.com/bbc/simorgh/blob/latest/docs/Coding-Standards/README.md)

[Repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)
